cloudhub-2/modules/ROOT/pages/ps-config-env.adoc

```diff
@@ -1,51 +1,82 @@
-= Associating Environments and Business Groups with a Private Space
+= Associating Business Groups and Environments with a Private Space
 
 Before deploying to a private space, you can specify environments and at least one business group to associate with it.
-This association specifies in which environments the private space appears as 
-a target deployment in the *Deployment Target* list when deploying an app. 
+This association specifies in which environments the private space appears as a target deployment in the *Deployment Target* list when deploying an app. 
 For information, see xref:ch2-deploy-private-space.adoc[].
 
-You can associate a private space with the production and sandbox environments
-in the business group, and its child business groups, that it's registered with or with a subset of business groups.
+You can associate a private space with the production and sandbox environments in the business group, and its child business groups, that it's registered with or with a subset of business groups.
 
 By default, a private space is associated with all sandbox environments in its business groups.
 
-== Associate Environments and Business Groups
+== Associate Business Groups and Environments
 
 If you have the CloudHub Network Administrator permission, associate environments in the business group or groups with the private space:
 
 //SELECT PRIVATE SPACE SHARED
 include::partial$select-private-space.adoc[tag=selectPrivateSpace]
 include::partial$select-private-space.adoc[tag=clickPrivateSpaceName]
-. Click the *Environments* tab.
-. Select the environments from the drop-down list:
+. Click the *Environments* tab to:
+** <<associate-business-groups>>
+** <<associate-environments>>
+
+=== Associate Business Groups
+
+. Select the business groups from the drop-down list:  
 +
 --
-** *Any environment*
-+
-Associates the private space with all sandbox and production environments in the business group and its children.
-** *Sandbox environments*
+** *This business group and its children*
++ 
+This option allows you to select either all environments, all sandbox environment or all production environments for the selected business group and all its children.
+** *Selected business groups*
 +
-Associates the private space with only sandbox environments in the business group and its children.
-** *Production environments*
+Enter the business group name in the *Add business group* field and select it from the list. This option allows you to select specific environments for specific child business groups.
```

Comment:
`select specific environments for specific child business groups` -> You can select specific environments for root org as well as it's children. So, can we change this to `specific business groups`